apps/src/templates/teacherNavigation/TeacherNavigationBar.tsx

```diff
@@ -60,71 +58,71 @@ const TeacherNavigationBar: React.FunctionComponent = () => {
     getSectionHeader(i18n.courseContent()),
     <SidebarOption
       icon={'desktop'}
-      optionTitle={i18n.course()}
       isSelected={false}
+      sectionId={+selectedSectionId}
+      pathKey={'courseOverview'}
     />,
     <SidebarOption
+      sectionId={+selectedSectionId}
       icon={'folder-open'}
-      optionTitle={i18n.lessonMaterials()}
-      isSelected={false}
-    />,
-    <SidebarOption
-      icon={'file-lines'}
-      optionTitle={i18n.lessonPlans()}
-      isSelected={false}
-    />,
-    <SidebarOption
-      icon={'presentation-screen'}
-      optionTitle={i18n.slideDecks()}
       isSelected={false}
+      pathKey={'lessonMaterials'}
     />,
     <SidebarOption
       icon={'calendar'}
-      optionTitle={i18n.calendar()}
       isSelected={false}
+      sectionId={+selectedSectionId}
+      pathKey={'calendar'}
     />,
   ];
 
   const performanceContent = [
     getSectionHeader(i18n.performance()),
     <SidebarOption
       icon={'chart-line'}
-      optionTitle={LABELED_TEACHER_NAVIGATION_PATHS.progress.label}
       isSelected={false}
+      sectionId={+selectedSectionId}
+      pathKey={'progress'}
     />,
     <SidebarOption
       icon={'star'}
-      optionTitle={LABELED_TEACHER_NAVIGATION_PATHS.assessments.label}
       isSelected={false}
+      sectionId={+selectedSectionId}
+      pathKey={'assessments'}
     />,
     <SidebarOption
       icon={'code'}
-      optionTitle={LABELED_TEACHER_NAVIGATION_PATHS.projects.label}
       isSelected={true}
+      sectionId={+selectedSectionId}
+      pathKey={'projects'}
     />,
     <SidebarOption
       icon={'chart-simple'}
-      optionTitle={LABELED_TEACHER_NAVIGATION_PATHS.stats.label}
       isSelected={false}
+      sectionId={+selectedSectionId}
+      pathKey={'stats'}
     />,
     <SidebarOption
       icon={'pen-line'}
-      optionTitle={LABELED_TEACHER_NAVIGATION_PATHS.textResponses.label}
       isSelected={false}
+      sectionId={+selectedSectionId}
+      pathKey={'textResponses'}
     />,
   ];
 
   const classroomContent = [
     getSectionHeader(i18n.classroom()),
     <SidebarOption
       icon={'users'}
```

Comment:
I'd like to consider adding the icon name to the `LABELED_TEACHER_NAVIGATION_PATHS` @lfryemason - thoughts on this? 

Pros:

- One spot to change icons in the future
- Means that we wouldn't need to pass it though here separately

Cons:

- Perhaps mucking up the constant...

Reply:
I love it